config/database.yml

```diff
@@ -4,23 +4,28 @@
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
 #
-default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
 
 development:
-  <<: *default
-  database: db/development.sqlite3
+  adapter: postgresql
```

Comment:
It's a good practice to define `default` config which includes common properties like `adapter`, `encoding` or `host` and include it in environment specific configs.